### PR TITLE
PDF improvements: names and symbols

### PR DESCRIPTION
**Pull request by williamdemeo**
# Description

This addresses all but two of the items in issue #481.  

The two outstanding items are the following:
- [x] [Lines 83--86](https://github.com/IntersectMBO/formal-ledger-specifications/blob/52e31136cdfb6e38130cacaed75392fece71c0a2/src/Ledger/Introduction.lagda#L82C1-L87C1) of `Introduction.lagda`: The symbol `X` is not defined.
- [x] [Line 211](https://github.com/IntersectMBO/formal-ledger-specifications/blob/52e31136cdfb6e38130cacaed75392fece71c0a2/src/Ledger/Utxo.lagda#L211) of `Utxo.lagda`: ∑[_←_] is not defined.

# Checklist

- [X] Commit sequence broadly makes sense and commits have useful messages
- [X] Code is formatted according to [CONTRIBUTING.md](https://github.com/input-output-hk/formal-ledger-specifications/blob/master/CONTRIBUTING.md)
- [X] Self-reviewed the diff

### CODE REVIEW

**Comment by williamdemeo** [comment]
Just two items left to do.

- [ ] [Lines 83--86](https://github.com/IntersectMBO/formal-ledger-specifications/blob/52e31136cdfb6e38130cacaed75392fece71c0a2/src/Ledger/Introduction.lagda#L82C1-L87C1) of `Introduction.lagda`: The symbol `X` is not defined.
- [ ] [Line 211](https://github.com/IntersectMBO/formal-ledger-specifications/blob/52e31136cdfb6e38130cacaed75392fece71c0a2/src/Ledger/Utxo.lagda#L211) of `Utxo.lagda`: ∑[_←_] is not defined.

**Review by WhatisRT — CHANGES_REQUESTED** [review]

File: src/Ledger/Utxow.lagda [hunk 1]
```diff
@@ -101,7 +101,7 @@ data _⊢_⇀⦇_,UTXOW⦈_ where
     ∙  neededHashes ＼ refScriptHashes ≡ᵉ witsScriptHashes
     ∙  inputHashes ⊆ txdatsHashes
     ∙  txdatsHashes ⊆ inputHashes ∪ allOutHashes ∪ getDataHashes (range (utxo ∣ refInputs))
-    ∙  txADhash ≡ map hash txAD
+    ∙  txADhash ≡ mapᴸ hash txAD
```
[the remark this answers is not in the document]
Comment (williamdemeo) [reply src/Ledger/Utxow.lagda:104]: I don't know how to fix this with a LaTeX macro. That's why I used Agda renaming.  Can you explain how you would do it with macros here or in the meeting today?

[the remark this answers is not in the document]
Comment (WhatisRT) [reply src/Ledger/Utxow.lagda:104]: Just do `\newunicodechar{ˢ}{}`.

File: src/Ledger/GovernanceActions.lagda [hunk 2]
```diff
@@ -154,9 +157,9 @@ protection. For \TreasuryWdrl and \Info, enacting them does not change
 the state in non-commutative ways, so they can always be enacted.
 
 Types related to this hash protection scheme are defined in
-Figure~\ref{fig:needshash-and-hashprotected-types}. \Unit is the unit
-type that has exactly one element, which reflects that a \GovActionID
-is not necessary.
+Figure~\ref{fig:needshash-and-hashprotected-types}. As mentioned,
+\Unit is the unit type that has exactly one element, which reflects
```
[the remark this answers is not in the document]
Comment (williamdemeo) [reply src/Ledger/GovernanceActions.lagda:161]: The sentence is not just re-explaining the unit.  It is also explaining *why* it's used here.

File: src/Ledger/Utxow.lagda (hunk 1 above)
[the remark this answers is not in the document]
Comment (williamdemeo) [reply src/Ledger/Utxow.lagda]: I tried the newunicodechar trick as well as DeclareUnicodeChar.  Neither worked for me, so I added another postprocessing python script which simply replaces each occurrence of `mapˢ` with `map`.

File: src/Ledger/GovernanceActions.lagda (hunk 2 above)
[the remark this answers is not in the document]
Comment (williamdemeo) [reply src/Ledger/GovernanceActions.lagda]: I've removed explanation of unit from `GovernanceActions` and added a more general description of that type to `Notation`.

File: src/Ledger/Utxow.lagda (hunk 1 above)
[the remark this answers is not in the document]
Comment (WhatisRT) [reply src/Ledger/Utxow.lagda]: I just put `\newunicodechar{ˢ}{\ensuremath{}}` and it works, so no need for another script.

[the remark this answers is not in the document]
Comment (williamdemeo) [reply src/Ledger/Utxow.lagda]: `\ensuremath{}` must insert the "missing" $ that latex was complaining about.  Good.

[the remark this answers is not in the document]
Comment (williamdemeo) [reply src/Ledger/Utxow.lagda]: Weird.  It works when I put it in `preamble.tex` but not when I put it in `iohk.sty`.

**Review by WhatisRT — APPROVED** [review]
LGTM

**Pull request merged**